install.sh

```diff
@@ -746,67 +761,121 @@ EOF
 fi
 
+if [ -z "$OVERWRITE_COLLECTD_CONFIG" ]; then
+    echo
+    echo "We recommend using Wavefront's collectd configuration for initial setup"
+    if ask "Would you like to overwrite any existing collectd configuration? " N; then
+        OVERWRITE_COLLECTD_CONFIG="yes"
+    else
+        echo
+        echo "The write_tsdb plugin is required to send metrics from collectd to the Wavefront Proxy"
+        echo "Manual setup is required"
+        echo
+    fi
+fi
+
+if [ -n "$OVERWRITE_COLLECTD_CONFIG" ]; then
+    hostname -f > /dev/null
+    if [ "$?" != 0 ]; then
+        exit_with_failure "Failed to resolve FDQN"
+    fi
+
+    if command_exists wget; then
+        FETCHER="wget --quiet -O /tmp/collectd_conf.tar.gz"
+    elif command_exists curl; then
+        FETCHER="curl -L --silent -o /tmp/collectd_conf.tar.gz"
+    else
+        exit_with_failure "Either 'wget' or 'curl' are needed"
+    fi
+    echo_step "  Configuring collectd"
+    $FETCHER https://github.com/wavefrontHQ/install/releases/download/1.0/collectd_conf.tar.gz >>${INSTALL_LOG} 2>&1
+    echo_success
+    echo_step "  Extracting Configuration Files"
+    if [ ! -d "/etc/collectd" ]; then
+        mkdir -p /etc/collectd
+    fi
+    tar -xf /tmp/collectd_conf.tar.gz -C /etc/collectd >>${INSTALL_LOG} 2>&1
+        if [ "$?" != 0 ]; then
+            exit_with_failure "Failed to extract configuration files"
+        fi
+    echo_success
+
+    case $OPERATING_SYSTEM in
+    REDHAT)
+        echo_step "  Overwriting collectd.conf"	
+        mv /etc/collectd.conf /etc/collectd.conf.bak
+        mv /etc/collectd/collectd.conf /etc/collectd.conf
+        echo_success
+        ;;
+    esac
+    
+    echo_step "  Modifying Configuration File at $COLLECTD_WAVEFRONT_CONF_FILE"
+    # Update the configuration file
+    sed -ri s,Host\\s+.*$,"Host \"$PROXY\"",g $COLLECTD_WAVEFRONT_CONF_FILE
+    sed -ri s,Port\\s+.*$,"Port \"$PROXY_PORT\"",g $COLLECTD_WAVEFRONT_CONF_FILE
+    echo_success
+    echo_step "  Restarting collectd"
+    service collectd restart >>${INSTALL_LOG} 2>&1
+    echo_success
+fi
+
+if [ -z "$APP_CONFIGURE" ]; then
+    echo
+    if ask "Would you like to configure collectd based on your installed app? " Y; then
+        APP_CONFIGURE="yes"
+    else
+        echo
+        echo "Keeping the default configuration"
+        echo
+    fi
+fi
+
+if [ -n "$APP_CONFIGURE" ]; then
+    if command_exists wget; then
+        FETCHER="wget --quiet -O /tmp/app_configure.tar.gz"
+    elif command_exists curl; then
+        FETCHER="curl -L --silent -o /tmp/app_configure.tar.gz"
+    else
+        exit_with_failure "Either 'wget' or 'curl' are needed"
+    fi
+    echo_step "  Pulling application configuration file"
+    $FETCHER https://github.com/kentwang929/install/files/348741/app_configure.tar.gz >>${INSTALL_LOG} 2>&1
+    echo_success
+    echo_step "  Extracting Configuration Files"
+    if [ ! -d "/tmp/app_configure" ]; then
+        mkdir -p /tmp/app_configure
+    fi
```

Comment:
I might name the directory with some sort of indicator that it's `wavefront` rather than just app.

Also, what would happen if that directory already exists with existing files?  Maybe you need to clear it?